src/mesh/Mesh.hpp

```diff
@@ -191,6 +191,13 @@ class Mesh {
   /// Computes the boundingBox for the vertices.
   void computeBoundingBox();
 
+  /// Computes if a quad is convex and returns and ordered set of vertices.
+  void computeQuadConvexityFromPoints(std::array<int,4> &hull) const;
```

Comment:
A more descriptive name than "hull" would be good.

Reply:
Would `vertexList` be better is it contains a list of vertix ID's?

Reply:
`vertexList` does not carry the information that those are IDs.
`vertexIDs`?